bc_obps/registration/api/user_operator.py

```diff
@@ -239,15 +236,18 @@ def create_user_operator_contact(request, payload: UserOperatorContactIn):
             senior_officer_contact.email = user.email
             senior_officer_contact.phone_number = user.phone_number
         else:
-            senior_officer_contact.first_name = payload_dict.get("first_name")
-            senior_officer_contact.last_name = payload_dict.get("last_name")
-            senior_officer_contact.email = payload_dict.get("so_email")
-            senior_officer_contact.phone_number = payload_dict.get("so_phone_number")
+            senior_officer_contact.first_name = payload.first_name
+            senior_officer_contact.last_name = payload.last_name
+            senior_officer_contact.email = payload.so_email
+            senior_officer_contact.phone_number = payload.so_phone_number
 
         senior_officer_contact.save()
 
     except ValidationError as e:
         return 400, {"message": generate_useful_error(e)}
+    # error handling when an existing contact with the same email already exists
+    except IntegrityError as e:
+        return 400, {"message": "A contact with this email already exists."}
```

Comment:
just FYI, not for this PR, Dylan said we should allow contacts with the same email so this is ultimately not going to be a good check

Reply:
Thanks for letting me know. In that case, we should also remove the `UniqueConstraint` on the contact data model.